exercises/ex29.js

```diff
@@ -0,0 +1,13 @@
+//Removing Duplicates
+let arr=[1,2,4,4,5,6,1];
+let arr1=myfunc(arr);
+document.write(arr1);
+function myfunc(arr){
+    let arr1=[];
+    for(let i=0;i<arr.length;i++){
+        if(!arr1.includes(arr[i])){
+            arr1.push(arr[i]);
+        }
+    }
+return arr1;
+}
```

Comment:
let a = ['A', 'B', 'A', 'C', 'B'];
let b = [...new Set(chars)]; 

try this to use code optimize. relate a large set of values.